src/app/search/search-workflow-table/search-workflow-table.component.ts

```diff
@@ -35,9 +35,12 @@ import { SearchService } from '../state/search.service';
 export class SearchWorkflowTableComponent extends SearchEntryTable implements OnInit {
   readonly entryType = 'workflow';
   public dataSource: MatTableDataSource<Workflow>;
-  public galaxyShortfriendlyName = this.descriptorLanguageService
-    .workflowDescriptorTypeEnumToShortFriendlyName(Workflow.DescriptorTypeEnum.Gxformat2)
-    .split(' ')[0];
```

Comment:
It looks like this `split` wasn't doing anything... `'gxformat2'.split(' ')` returns the array `['gxformat2']`.

You can remove the comment then if you want, but either way. I was trying to understand why there was a `split()` there, which is why I made my request for a comment. But since you now removed the `split()`, the code is now pretty self-explanatory.

Reply:
I just left the comment in